release/preferences.py

```diff
@@ -0,0 +1,85 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Access and manage name&value preferences, persisted in a local JSON file.
+"""
+
+import os
+import json
+
+THIS_DIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
So this moves the location of this file. Committers should move their existing file in the release folder now?

Reply:
That should work. But I'd rather if committers start anew with this file. There are new preference keys with this change, but the full list of questions isn't cumbersome enough to go through to warrant reusing the old file, and I didn't actually test compabitibility.